.github/workflows/superset-python-integrationtest.yml

```diff
@@ -150,7 +150,9 @@ jobs:
       SUPERSET_CONFIG: tests.integration_tests.superset_test_config
       REDIS_PORT: 16379
       SUPERSET__SQLALCHEMY_DATABASE_URI: |
-        sqlite:///${{ github.workspace }}/.temp/unittest.db
+        sqlite:///${{ github.workspace }}/.temp/superset.db?check_same_thread=true
```

Comment:
This threw me for a curveball. I was able to successfully run the tests locally but they failed remotely with a somewhat cryptic error stating a reference table being specified in [this](https://github.com/apache/superset/blob/3a7d76cece4408c8e80862ab9db3a38eaaafdc8c/tests/integration_tests/datasets/api_tests.py#L140-L160) fixture didn't exist.

I speculate that this is probably due to a multithreading or connection pooling issue when running via a GitHub workflow. Ensuring a single connection per thread _seems_ to remedy the problem.